app/(pages)/(home)/page.tsx

```diff
@@ -151,6 +150,7 @@ export default function Home() {
                 // target="_blank"
                 // rel="noreferrer"
                 className={cn(buttonVariants(), "px-4")}
```

Comment:
you can just add className along with `className={cn(buttonVariants(), "px-4")}`

like `className={cn(buttonVariants(), "px-4 tailwind-class-you-wan-to-add")}`

Reply:
Okay